### PR TITLE
Adds notes for 4.10.14 and console feature

### PR DESCRIPTION
**Pull request by opayne1**
Adds notes for 4.10.14 release and adds console feature for updating clusters

- [OSDOCS-3584](https://issues.redhat.com/browse/OSDOCS-3584)

Version(s):
Applies to `enterprise-4.10` only

Link to docs preview:
[Preview link](https://deploy-preview-45730--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes.html#ocp-4-10-14)

Additional information:
Relates to:[ #11053](https://github.com/openshift/console/pull/11053)

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -2866,3 +2866,29 @@ link:https://access.redhat.com/solutions/6956858[{product-title} 4.10.13 contain
 ==== Updating
 
 To upgrade an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-10-14"]
+=== RHBA-2022: - {product-title} 4.10.14 bug fix update
```
Comment (jeana-redhat) [inline release_notes/ocp-4-10-release-notes.adoc:2871]:
```suggestion
=== RHBA-2022:2178 - {product-title} 4.10.14 bug fix update
```

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -2866,3 +2866,29 @@ link:https://access.redhat.com/solutions/6956858[{product-title} 4.10.13 contain
...
+
+link:https://access.redhat.com/solutions/6958424[{product-title} 4.10.14 container image list]
+
+[id="ocp-4-10-14-features"]
+==== Features
+
+[id="ocp-4-9-7-update-mode-update-cluster"]
+===== Update the control plane independently of other worker nodes
+
+With this update, you are now able to do a partial cluster update within the *Update Cluster* modal. You are able to update the worker or custom pool Nodes to accommodate the time it takes for maintenance. You can also pause and un-pause within each pool's progress bar. An alert will display at the top of the *Cluster Settings* page if one or more worker or custom pools are paused. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076777[*BZ#2076777*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-10-release-notes.adoc:2887]:
```suggestion
With this update, you can now perform a partial cluster update within the *Update Cluster* modal. You are able to update the worker or custom pool nodes to accommodate the time it takes for maintenance. You can also pause and resume within the progress bar of each pool. If one or more worker or custom pools are paused, an alert is displayed at the top of the *Cluster Settings* page . (link:https://bugzilla.redhat.com/show_bug.cgi?id=2076777[*BZ#2076777*])
```

File: release_notes/ocp-4-10-release-notes.adoc [hunk 3]
```diff
@@ -2866,3 +2866,29 @@ link:https://access.redhat.com/solutions/6956858[{product-title} 4.10.13 contain
...
+For more information, see ...
```
Comment (jeana-redhat) [inline release_notes/ocp-4-10-release-notes.adoc:2888]: TBD looks like

**Review by jeana-redhat — COMMENTED** [review]
Small suggestions on that new feature text

File: release_notes/ocp-4-10-release-notes.adoc (hunk 3 above)
Comment (opayne1) [reply release_notes/ocp-4-10-release-notes.adoc:2888]: Yes may be one or 2 xrefs. Not sure yet.

**Comment by opayne1** [comment]
@rhamilto PTAL :)

File: release_notes/ocp-4-10-release-notes.adoc (hunk 3 above)
Comment (rhamilto) [reply release_notes/ocp-4-10-release-notes.adoc:2888]: LGTM

**Comment by opayne1** [comment]
@yapei This is the release note for the partial upgrade that is going in with 4.10.14. Could I have your ack on this? Thank you!

**Pull request merged**